ui/templates/footer.html

```diff
@@ -1,25 +1,75 @@
+                <span class="rrssb-text">facebook</span>
+              </a>
+            </li>
+
+            <li class="rrssb-twitter">
+              <a href=""
+                 class="popup">
+                <span class="rrssb-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28">
+                  <path d="M24.253 8.756C24.69 17.08 18.297 24.182 9.97 24.62a15.093 15.093 0 0 1-8.86-2.32c2.702.18 5.375-.648 7.507-2.32a5.417 5.417 0 0 1-4.49-3.64c.802.13 1.62.077 2.4-.154a5.416 5.416 0 0 1-4.412-5.11 5.43 5.43 0 0 0 2.168.387A5.416 5.416 0 0 1 2.89 4.498a15.09 15.09 0 0 0 10.913 5.573 5.185 5.185 0 0 1 3.434-6.48 5.18 5.18 0 0 1 5.546 1.682 9.076 9.076 0 0 0 3.33-1.317 5.038 5.038 0 0 1-2.4 2.942 9.068 9.068 0 0 0 3.02-.85 5.05 5.05 0 0 1-2.48 2.71z"/></svg></span>
+                <span class="rrssb-text">twitter</span>
+              </a>
+            </li>
+            <li class="rrssb-googleplus">
+              <a href="" class="popup">
+                <span class="rrssb-icon">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+                    <path d="M21 8.29h-1.95v2.6h-2.6v1.82h2.6v2.6H21v-2.6h2.6v-1.885H21V8.29zM7.614 10.306v2.925h3.9c-.26 1.69-1.755 2.925-3.9 2.925-2.34 0-4.29-2.016-4.29-4.354s1.885-4.353 4.29-4.353c1.104 0 2.014.326 2.794 1.105l2.08-2.08c-1.3-1.17-2.924-1.883-4.874-1.883C3.65 4.586.4 7.835.4 11.8s3.25 7.212 7.214 7.212c4.224 0 6.953-2.988 6.953-7.082 0-.52-.065-1.104-.13-1.624H7.614z"/></svg>            </span>
+                <span class="rrssb-text">google+</span>
+              </a>
+            </li>
+            <li class="rrssb-linkedin">
+              <a href="" class="popup">
+                <span class="rrssb-icon">
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28">
+                    <path d="M25.424 15.887v8.447h-4.896v-7.882c0-1.98-.71-3.33-2.48-3.33-1.354 0-2.158.91-2.514 1.802-.13.315-.162.753-.162 1.194v8.216h-4.9s.067-13.35 0-14.73h4.9v2.087c-.01.017-.023.033-.033.05h.032v-.05c.65-1.002 1.812-2.435 4.414-2.435 3.222 0 5.638 2.106 5.638 6.632zM5.348 2.5c-1.676 0-2.772 1.093-2.772 2.54 0 1.42 1.066 2.538 2.717 2.546h.032c1.71 0 2.77-1.132 2.77-2.546C8.056 3.593 7.02 2.5 5.344 2.5h.005zm-2.48 21.834h4.896V9.604H2.867v14.73z"/></svg>
+                </span>
+                <span class="rrssb-text">linkedin</span>
+              </a>
+            </li>
+
+          </ul>
+        </div>
+        <div class="footer-copy">&copy; 2016, Massachusetts Institute of Technology</div>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    var url='{{ request.build_absolute_uri }}';
```

Comment:
Are you setting this `url` variable so that `public.js` can reference it? If so, I recommend two things:

* Move this to `templates/base.html`, where the `SETTINGS` variable is defined. We should have as few places as possible where global variables are set; ideally, that should be the *only* place.
* Pick a more descriptive name than `url`. That variable name is just begging to be overwritten by some other temporary `url` variable in some other script. I would recommend `currentPageUrl` or `CURRENT_PAGE_URL`.